client/src/components/Projects/ProjectNew/index.jsx

```diff
@@ -34,26 +57,50 @@ export default class index extends Component {
     e.preventDefault();
     const {
       selection: { row },
-      newProject: { name }
+      newProject: { name, dsescription }
     } = this.state;
     if (name.trim().length === 0) {
       //show error here for project name
       this.setState({
         error: {
           errorStatus: true,
-          errorMsg: "Please enter the project name"
+          errorMsg: 'Please enter the project name'
         }
       });
     } else if (!Array.isArray(row) || row.length === 0) {
       //show error here for selection
       this.setState({
         error: {
           errorStatus: true,
-          errorMsg: "Please select at least one team member"
+          errorMsg: 'Please select at least one team member'
```

Comment:
I think it is better to make a separate function to validate project data and call it in handleOnChange. This will make your code more organized

Reply:
Also it is not mine!!! 
We will do all refactoring after we finish
Thanks

Reply:
@IsraaSulaiman  and @AymanAlqoqa 
we all should have reviewed it before, so it's our mistake all of us
`it's not my mistake` isn't good for teamwork

anyway, I agree with @AymanAlqoqa  we should do the refactor for the next week
today we need to finish up something working to deliver for our client tomorrow
plus don't resolve a conversation that you didn't start :wink: